packages/stencil/components/tabs/src/components/osds-tab-bar-item/osds-tab-bar-item.scss

```diff
@@ -0,0 +1,63 @@
+@import '~@ovhcloud/ods-theming/ods-theme';
+@import '../osds-tabs/styles/osds-tabs.typography.scss';
+@import '../osds-tabs/styles/osds-tabs.color.scss';
+@import '~@ovhcloud/ods-theming/color/ods-theming-color';
+@import '~@ovhcloud/ods-theming/ods-theme';
+
+.tabs-tab {
+    position: relative;
+    display: inline-flex;
+    align-items: center;
+    outline: none;
+    cursor: pointer;
+
+    &.tabs-tab-contrasted {
+
+      div:hover {
+        border-bottom: 2px solid var(--ods-color-primary-100);
```

Comment:
- please add style regarding colors in component color file
- you can also use some mixin here I think
- check Button example for how we deal with applying css to attributes, here contrasted &[contrasted] {} : https://github.com/ovh/design-system/blob/master/packages/stencil/components/button/src/components/osds-button/styles/osds-button.color.scss#L16